fe/src/main/java/org/apache/doris/http/common/DorisHttpObjectAggregator.java

```diff
@@ -0,0 +1,62 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.http.common;
+
+import io.netty.channel.ChannelPipeline;
+import io.netty.handler.codec.http.HttpContent;
+import io.netty.handler.codec.http.HttpMessage;
+import io.netty.handler.codec.http.HttpObject;
+import io.netty.handler.codec.http.HttpObjectAggregator;
+import io.netty.handler.codec.http.HttpRequest;
+import io.netty.handler.codec.http.HttpUtil;
+
+/*
+ * don't handle 100-continue and chunked transfer-encoding http header
+ */
+public class DorisHttpObjectAggregator extends HttpObjectAggregator {
+    // the flag for aggregator whether has started
+    // in order not to handle chunked transfer-encoding header in {@link isContentMessage} method
```

Comment:
`HttpObjectAggregator.beginAggregation` will remove chunked transfer-encoding header.

`HttpObjectAggregator.finishAggregation`  will set content-length header.

Reply:
what problem will it cause that aggregator replaces chucked with content-length? 
I think request body is aggregated and pass to handler, and it is OK.

Reply:
I think if there is 100-continue header, FE returns redirect to client, then client will send request to BE, and the headers which be received is sent from client not from FE.

And if there is no 100-continue header, FE will reject this request. 

If all loading data is routed by FE, it is a huge load for it. So we need client must add "100-continue" header to request's header, and then FE can redirect the client to some BE to handle this load request

Reply:
Yes. If we want to get request body, we need to aggregate it. but for the `LoadAction`, we shouldn't let FE handle the Load data. which is conflicting. So, I think we could make `DorisHttpObjectAggregator` only handle post request. what do you think of it ?

Reply:
The followings are what I want to figure out for current `DorisHttpObjectAggregator`

1. If request has 100-continue header,  does it receive request body before call action's handler? If this is true, I think in normal case for `LoadAction`, FE won't receive load data.

2. If request has no 100-continue header and request body is exceed max body size, will it return an error response to client ? If this is true, I think it's OK to receive relatively small data, however it is difficult to let client known that request misses 100-continue header, because aggregator is a common path for all input HTTP request.

Making `DorisHttpObjectAggregator` only apply POST request seems OK.

Reply:
1 Yes.  `DorisHttpObjectAggregator`  receive all request body before call  `HttpServerHandler` action's handler. 

2 Yes.  `DorisHttpObjectAggregator`  will response `413 Request Entity Too Large`.

Reply:
OK. I see.  I agree with you.